packages/braid-design-system/src/lib/components/PasswordField/PasswordField.tsx

```diff
@@ -76,11 +76,13 @@ export const PasswordField = forwardRef<HTMLInputElement, PasswordFieldProps>(
       [visible, onVisibilityToggle, inputRef],
     );
 
+    // Todo - remove once `ButtonIcon` id prop is optional
+    const id = useId();
```

Comment:
I would suggest making this more specific, e.g. `visibilityToggleId`, and remove the templating below. The templating was only to make it unique off a single input value.
```suggestion
    const visibilityToggleId = useId();
```